app/src/examples/WithClampExample.tsx

```diff
@@ -27,23 +29,47 @@ function renderExample(testedStyle: ViewStyle, description: string) {
           borderWidth: BORDER_WIDTH,
           borderColor: VIOLET,
         }}>
-        <Animated.View
-          style={[
-            styles.clampMarker,
-            { marginBottom: -CLAMP_MARKER_HEIGHT, width: UPPER_BOUND },
-          ]}
-        />
+        <View>
```

Comment:
Do we need this wrapping view? Can we remove it or use `<>` instead?

Reply:
We can't use `<>` instead, this view here is needed for absolute styling,